src/guesswidth.rs

```diff
@@ -190,12 +185,10 @@ fn split(line: &str, pos: &[usize], trim_space: bool) -> Vec<String> {
         w += UnicodeWidthStr::width(lr[p].to_string().as_str());
     }
 
-    if n <= columns.len() {
```

Comment:
I think the `if` condition is useless.

Because `n` is always added with `columns.push`, so n always less or equal to `columns.len()`